README.md

```diff
@@ -96,143 +65,95 @@ After you're finished please remove all the comments and instructions!
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
+ 
 
-> Describe between 1-3 key features of the application.
+- **Create a budget app**
+- **register and log in, so that the data is private to them.** 
+- **introduce new transactions associated with a category.**
+- **see the money spent on each category.**
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right"><a href="#readme-top">back to top</a></p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
+ 
+- [Live Demo Link](https://budget-app-ybru.onrender.com/)
 
-> Add a link to your deployed project.
+- [Video Demo Link](https://drive.google.com/file/d/1nzuq6zkgsB7Z6hqGqdmswTAGJ8MZlpFH/view?usp=sharing)
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right"><a href="#readme-top">back to top</a></p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
+**To get a local copy up and running, follow these steps.**
+1. Open terminal or cmd on you local machine and navigate to your desired directory.
+2. **Run**    *'git clone clone https://github.com/AbigiyaTY/Budget_app.git'*
+3. **Run**   *'code .'*
 
 ### Prerequisites
 
-In order to run this project you need:
 
-<!--
-Example command:
+In order to run this project you need **[describe prerequisites]**:
 
 ```sh
  gem install rails
 ```
- -->
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
 ```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone https://github.com/AbigiyaTY/Budget_app.git
+  cd Budget_app
 ```
---->
 
 ### Install
 
-Install this project with:
-
-<!--
-Example command:
+Install this project with **[package manager]**:
 
 ```sh
-  cd my-project
+  cd Budget_app
   gem install
+  rails db:migrate
+  rails db:migrate RAILS_ENV=test
 ```
---->
 
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
   rails server
```

Comment:
- [ ] OPTIONAL
Similarly, it's good idea to add instructions on how to run test cases for your app.